### PR TITLE
feat: #1419 - make value nullable for Recovery address

### PR DESCRIPTION
**Pull request by dibyajyotibehera**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md).
- [ ] I am following the
      [contributing code guidelines](../blob/master/CONTRIBUTING.md#contributing-code).
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security. vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added or changed [the documentation](docs/docs).

### CODE REVIEW

File: persistence/sql/migrations/templates/20210825184606_recovery_addresses.down.fizz [hunk 1]
```diff
@@ -0,0 +1 @@
+change_column("identity_recovery_addresses", "value", "string", {"size": 400,"null": false})
```
Comment (aeneasr) [inline persistence/sql/migrations/templates/20210825184606_recovery_addresses.down.fizz:1]: Fizz has a bug where `null: false` doesn't work, see https://github.com/gobuffalo/fizz/issues/62

```suggestion
change_column("identity_recovery_addresses", "value", "string", {"size": 400})
```

**Review by aeneasr — COMMENTED** [review]
just fyi :)

**Comment by aeneasr** [comment]
As per discussion on slack, I believe this is not the best approach to solve the recovery without email issue. Should we keep this open or do you want to close it?

**Comment by dibyajyotibehera** [comment]
Yes i will close it now

**Pull request closed without merging**